README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider adding a check to see if Homebrew is installed before attempting to install 'go-task' to make the script more robust.